src/uint/boxed/encoding.rs

```diff
@@ -131,6 +131,40 @@ impl BoxedUint {
 
         out.into()
     }
+
+    /// Create a new [`BoxedUint`] from the provided big endian hex string.
+    pub fn from_be_hex(hex: &str, bits_precision: u32) -> Self {
```

Comment:
Since these can't be constructed at compile-time, it would probably be good to make it fallible